package.json

```diff
@@ -34,11 +34,13 @@
         "build:prod": "jlpm clean && jlpm build:lib:prod && jlpm build:labextension",
         "build:labextension": "jupyter labextension build .",
         "copy:landing:template": "cp templates/index.html dist/index.html",
+        "copy:favicon": "cp static/favicon.ico dist/favicon.ico && cp static/favicon.ico dist/lab/favicon.ico",
+        "fix:lab:title": "sed 's/<title>JupyterLite<\\/title>/<title>Jupyter Everywhere<\\/title>/g' dist/lab/index.html > dist/lab/index.html.tmp && mv dist/lab/index.html.tmp dist/lab/index.html",
```

Comment:
Should we  have an integration test for this? It should be three lines or so:

```ts
expect(page).toHaveTitle('Jupyter Everywhere');
```

I just worry this might break easily if we modify dist files.

Reply:
I had initially left doing this, but I shall add it if you insist. We only have two places where this is used, so I hope it won't break frequently.

Reply:
Done in 1d430590296fa9050de1ad4e1110abe7e0939d20